### PR TITLE
Feature/separate inventory

### LINKED ISSUE #159: Separate Inventory Into Projects vs Tools vs Apps
In order to promote the practice that projects are a more privileged object. Should also update labels on templates to make it easier to `oc delete all -l foo=bar` instead of `oc delete project`.

This also means moving app config to `.openshift-applier` directories in app source code, so we'll need to address the issue of how you stand up labs-ci-cd with those apps for things like CI, since we need an app to go through the pipeline for regression testing.

### PR DESCRIPTION
**Pull request by springdo**
@sherl0cks  - one for you when you get a chance. Probs will be some back and forth on this one. I have moved the `app` stuff into a separate inventory too as an intermediate step before we completely remove it / think of a better house for it ie inside the java-app's own `.openshift-applier` dir. 

We could also just get rid of it completely ;)

closes #159

### CODE REVIEW

**Comment by oybed** [comment]
@springdo @sherl0cks Not sure I totally agree with moving away from the `openshift-applier` provided playbook. Maybe we can discuss more details when I'm back from vacation.

File: params/projectrequests/ci-cd [hunk 1]
```diff
@@ -1,2 +1,2 @@
 NAMESPACE=labs-ci-cd
-NAMESPACE_DISPLAY_NAME=Labs CI/CD
+NAMESPACE_DISPLAY_NAME=donal CI/CD
```
[the remark this answers is not in the document]
Comment (springdo) [reply params/projectrequests/ci-cd]: Good spot! I did a search and replace on `donal-` which didn't catch those ones!

**Comment by springdo** [comment]
@oybed  - I've updated as per your change req! Not sure what you mean about moving away from the `openshift-applier`..... this change all still uses the `openshift-applier` to drive cluster content; just in separate inventory files for different type of operations

**Comment by oybed** [comment]
@springdo I should have bolded the `openshift-applier` **provided playbook**  portion.  Basically, I'm thinking some of the changes made here may be a good change in the `openshift-applier` repo's provided playbook. But again, we can discuss that when I'm back from vacation to be more specific.

**Comment by springdo** [comment]
@oybed what changes were you thinking in particular of moving over? Trying to figure it out as don't want to have to sit on a PR for 2 weeks before we make any progress. This is the first in a few big changes we want to make to this repo so best bottom them out earlier!

**Comment by oybed** [comment]
@springdo - my comment isn’t as easy as just a quick move-a-few-lines and be done. I’d say go ahead and merge when you guys are good with this PR and we can address the playbook comment later. It also requires completing a few other outstanding features for openshift-applier, so addressing it later makes more sense from that perspective as well.

**Comment by pcarney8** [comment]
@oybed @springdo i would also like to talk about this also when you get back.

**Comment by pcarney8** [comment]
@springdo not really any changes. i'm just whole hearted-ly in support of using the `apply.yml` and making the applier a little more Ansible-like and Ansible-friendly. The multiple inventories + multiple playbooks came out of some work I did in January and then I believe Justin did some cleaning up into the `host_vars` so I just wanted to get my support for it out there in support of what you're doing :smile: 

I'll have a follow up on this PR that will allow the project requests be a little more dynamic. But we can discuss that later

**Comment by springdo** [comment]
@pcarney8  cool man! I think there is a lot of _follow on_ work once this PR gets in. I'm guessing this is one of the higher priority PRs? Im at enablement next week so won't have time to focus on any of these PRs. Just thinking we'll probs need to update the CI/CD for the Labs CI/CD project to handle multiple inventory though

**Comment by sherl0cks** [comment]
Just a note that it will be at least 1 week before I get cycles to look at this. Candidly, if we have a reasonable CI strategy and a plan on how to move forward with multiple inventories, I'm OK with other team members closing this out without my direct input

File: inventory/host_vars/ci-cd-tooling.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  - name: nexus
+    template: "{{ playbook_dir }}/openshift-templates/nexus/template.json"
+    params: "{{ playbook_dir }}/params/nexus/deploy"
+    namespace: "{{ ci_cd_namespace }}"
+    tags:
+    - nexus
+    post_steps:
+    - role: "labs-ci-cd/roles/configure-nexus"
+  - name: sonarqube
+    template: "{{ playbook_dir }}/openshift-templates/sonarqube/template.json"
```
Comment (oybed) [inline inventory/host_vars/ci-cd-tooling.yml]: make sure this is fixed per PR https://github.com/rht-labs/labs-ci-cd/pull/175

Comment (springdo) [reply inventory/host_vars/ci-cd-tooling.yml]: Cheers @oybed; I have some merge conflicts from all the other changes anyway so was watching for this one too 👍

**Comment by makentenza** [comment]
Also please point this URL on the general README file to the latest link.

FROM: https://docs.openshift.com/container-platform/3.7/cli_reference/get_started_cli.html

TO: https://docs.openshift.com/container-platform/latest/cli_reference/get_started_cli.html

**Comment by makentenza** [comment]
LGTM now

**Pull request merged**